rex/utilities/bc_utils.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+rex bias correction utilities.
+"""
+import json
+import numpy as np
+import logging
+from warnings import warn
+import rex.bias_correction
+
+
+logger = logging.getLogger(__name__)
+
+
+def _parse_bc_table(bc_df, gids):
```

Comment:
Is this intentionally private? In the docs, `bc_utils` will look like an empty module

Reply:
Good point. Its a tricky one because it causes a recursive import. I moved it to a new .py file, made it public, and added a real bc_utils.py file with actual utilities that can be imported by bias correction